### PR TITLE
prepping view user and laravel mix

### PR DESCRIPTION
**Pull request by kienpham2004**
prepping view user and laravel mix

### CODE REVIEW

**Comment by kienpham2004** [comment]
ready

File: resources/views/content.blade.php [hunk 1]
```diff
@@ -0,0 +1,53 @@
+@extends('layouts.app')
+@section('content')
+
+  <div class="header">
+  <a href="{{ route('user.index') }}" class="logo">Laravel Quicktask</a>
```
Comment (nguyenhuydat98) [inline resources/views/content.blade.php:5]: indent 4 space

File: resources/views/layouts.blade.php/app.blade.php [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        <!-- Fonts -->
+        <link href="https://fonts.googleapis.com/css2?family=Nunito:wght@200;600&display=swap" rel="stylesheet">
+        
+        <!-- Styles -->
+        <link rel="stylesheet" href="{{ asset('css/app.css') }}">
+    </head>
+    <body>
+    @include('sweetalert::alert')
+    @yield('content')
+    </body>
```
Comment (TDM1911) [inline resources/views/layouts.blade.php/app.blade.php:22]:
```suggestion
    <body>
        @include('sweetalert::alert')
        @yield('content')
    </body>
```

File: resources/views/userindex.blade.php [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        </tr>
+      </thead>
+     <tbody>
+      @foreach ($users as $user)
+      <tr>
+        <td>{{$loop->index}}</td>
+        <td>{{$user->name}}</td>
+        <td>{{$user->email}}</td>
+        <td>{{$user->created_at}}</td>
+        <td>{{$user->updated_at}}</td>
```
Comment (TDM1911) [inline resources/views/userindex.blade.php:26]:
```suggestion
        <td>{{ $loop->index }}</td>
        <td>{{ $user->name }}</td>
        <td>{{ $user->email }}</td>
        <td>{{ $user->created_at }}</td>
        <td>{{ $user->updated_at }}</td>
```

File: resources/views/content.blade.php [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+        <a href="{!! route('user.lang', ['vi']) !!}" >Tiếng Việt</a>
+      </div>
+  </div>
+
+  <div class="sidenav sidea">
+    <a href="{{ route('user.index') }}">User list</a>
+    <a href="{{ route('order.index') }}">Order list</a>
+  </div>
+
+  <div class="showindex" >
```
Comment (huyenttm-0901) [inline resources/views/content.blade.php:17]: indent 4 spaces

File: resources/views/content.blade.php [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+      <div class="header-right">
+        <a href="{!! route('user.lang', ['en']) !!}" >English</a>
+        <a href="{!! route('user.lang', ['vi']) !!}" >Tiếng Việt</a>
+      </div>
+  </div>
+
+  <div class="sidenav sidea">
+    <a href="{{ route('user.index') }}">User list</a>
+    <a href="{{ route('order.index') }}">Order list</a>
+  </div>
```
Comment (huyenttm-0901) [inline resources/views/content.blade.php:15]: i18n

File: resources/views/layouts.blade.php/app.blade.php [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
+        <script src="https://unpkg.com/sweetalert2@7.18.0/dist/sweetalert2.all.js"></script>
+
+        <title>Quicktask</title>
+
+        <!-- Fonts -->
+        <link href="https://fonts.googleapis.com/css2?family=Nunito:wght@200;600&display=swap" rel="stylesheet">
```
Comment (huyenttm-0901) [inline resources/views/layouts.blade.php/app.blade.php:14]: ko dùng link online, dùng bower nhé

File: resources/views/usercreate.blade.php [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+<form class="form-horizontal" action="{{ route('user.store') }}" method="POST">
+  @csrf
+    <fieldset>
+    <h2 class="adduser">{{ trans('trans.add_user') }}</h2>
+   
+    <div class="form-group">
+      <label for="name">{{ trans('trans.username') }}</label>
+      <input type="text" class="form-control" id="email" 
+              placeholder="{{ trans('trans.plh_name') }}" name="name" 
+               value="{{ old('name') }}" >
```
Comment (huyenttm-0901) [inline resources/views/usercreate.blade.php:13]: indent 4 spaces

**Pull request closed without merging**